webapp/src/dogma/common/components/Navbar.tsx

```diff
@@ -123,8 +148,11 @@ export const Navbar = () => {
             onChange={(option) => handleChange(option as ProjectOptionType)}
             placeholder="Jump to project ..."
             closeMenuOnSelect={true}
+            openMenuOnFocus={true}
```

Comment:
Suggestion: Could we set a different color such as a darker one to the background color of the select input to emphasize it?

Reply:
Otherwise, it is also good to set a distinguishable `borderColor` to `container` of the select box. 

I'm not 100% sure it is a good suggestion. 😅 UX and design are conundrums to me.

Reply:
Updated to the following.
<img width="442" alt="Screen Shot 2022-12-16 at 10 59 32" src="https://user-images.githubusercontent.com/5079588/208019070-5f8a1522-fe2d-4108-84c7-8beadeac5801.png">

Reply:
Awesome!
By the way, `X` button is shown on the right without the initial placeholder when a page is newly rendered.
<img width="840" alt="image" src="https://user-images.githubusercontent.com/1866157/208027468-d646a36a-8498-48e9-8620-251d3dde458f.png">

Reply:
🙇  Fixed.